src/resources/wkConfig.yaml

```diff
@@ -74,6 +74,8 @@ kind: ConfigMap
 metadata:
   name: watch-keeper-non-namespaced
   namespace: "{{ desired_namespace }}"
+data:
+  v1_node: lite
```

Comment:
Do we want to make this cm EnsureExists so we don't overwrite user customizations?

Reply:
it is https://github.com/razee-io/razeedeploy-delta#ensure-exist-resources